examples/tables.ipynb

```diff
@@ -67,7 +102,73 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Most tables are stored during the running of an fmu run. So you will in that case have one version per realization (i.e per file path realization-n/iter-m) <br>\n",
+    "This means that it will be a slow process to get to results from one ensemble, meaning one iteration. <br>\n",
+    "Below is an example of how to fetch one table. General syntax: <br>\n",
+    "``table_collection = case.tables.filter(realization=<realization number>, iteration=<iteration name>,``<br>\n",
+    "&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;``name=<table name>, tagname=<name of tag>)``<br>\n",
+    "Any of the input arguements are optional, so you could end up with more than one resulting table, and in theory even if you have <br>\n",
+    "used all arguements you could end up with several (long story, please ask). But you can check how many tables you have with using <br>\n",
+    "``len(table_collection)``<br>\n",
+    "To get to one table you can then access them using indexing as you would with a list in python: <br>\n",
+    "``table = table_collection[0]``<br>\n",
+    "If you want to access the actual object, you can access this with the methods to_pandas or to_arrow, and you also have access to the <br>\n",
+    "metadata via indexing as you would do a dictionary. The entire metadata can be access with using the attribute _metadata\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Filter using the key\n",
+    "one_table = tables.filter(realization=0, iteration=\"iter-0\", name=\"DROGON\", tagname=\"compdat\")[0]\n",
+    "# fetching the actual table as a pandas dataframe\n",
+    "print(one_table.name)\n",
+    "print(one_table.to_pandas.head())\n",
+    "# Access to the metadata\n",
+    "print(one_table[\"data\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you know the metadata you can access the data directly on the object with square brackets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "one_table[\"data\"].keys()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since it is slow to aggregate the tables yourself, and that the tables then would become very large Sumo comes with a service that aggregates the <br>tables, and then splits them up by column. We refer to these objects as aggregated tables even though they are both aggregated and split up.<br> These come in different types, but the most general is the collection, where you get all realizations stacked on top of each other,<br> but you also have access to statistical aggregations such as mean, min,max, std, p10, and p90 <br>\n",
+    "\n",
```

Comment:
Do we need/want to expose the fact that the aggregated tables are split up to users?

Reply:
I think we should, but have modified the text to clarify a bit around this.